drivers/project_pass_fail/app/models/project_pass_fail/project_pass_fail.rb

```diff
@@ -196,9 +196,7 @@ def timeliness_threshold
     end
 
     private def hmis_projects
-      query = GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids)
-      query = query.with_project_type(filter.project_type_ids) if filter.project_type_ids.any?
-      query
```

Comment:
I'm not convinced this change is correct.  Previously this would further limit the selection to only those in the selected project types, now it's adding anything from the project types selected.  Replacing with `filter.anded_effective_project_ids` _might_ fix it.

Reply:
The old behavior was incorrect -- Karmyn was trying to run a report for TH projects, and it was getting filtered out...

Looking at past reports, narrowing for project type looks like it **might** be what they expect -- So, I'm going to apply the filter _unless_ the default project types are what is selected.